en_US/faq/use-guide.md

```diff
@@ -417,3 +417,141 @@ Client disconnect link error code list:
+
+EMQX provides an environment variable `EMQX_HOST`, which allows us to set the Host part of the node name. Of course, the premise is that this Host must be connectable to other nodes, so we also need to use it with the network alias:
+
+```
+docker run -d --name emqx -p 18083:18083 -p 1883:1883 -e EMQX_HOST=alias-for-emqx --network example --network-alias alias-for-emqx --mount type=bind,source=/tmp/emqx,target=/opt/emqx/data emqx:5.0.24
+```
+
+## Reasons for common logs
+
+**Tags:** [*Log*](tags.md#log)
+
+### Keyword: “reason: {shutdown, takeovered}”
+
+In EMQX, each MQTT client connection is maintained by an independent process. Whenever an MQTT client connects, EMQX will create a new process. If the client wants to restore communication from the existing session state, EMQX needs to migrate the session state from the old process to the new one. We call this process: Take over, and the old process will be closed after the takeover is completed, and the reason for closing is `{shutdown, takeovered}`. If the original client is still connected, EMQX will send a DISCONNECT packet with Reason Code 0x8E (Session taken over), and then close the old connection.
+
+### Keyword: “reason: {shutdown, discarded}”
+
+Contrary to session takeover, if the client indicates that it wants to start a new session when connecting, then EMQX needs to discard the old session state, and EMQX will close the old process with the reason `{shutdown, discarded}`. If the original client is still connected, EMQX will send a DISCONNECT packet with Reason Code 0x8E (Session taken over), and then close the old connection.
+
+### Keyword: “reason: {shutdown, kicked}”
+
+Indicates that the client connection was manually kicked out (click the Kick Out button on the Dashboard or call the `DELETE clients/{clientid}` API). If the original client is still connected, EMQX will send a DISCONNECT packet with Reason Code 0x98 (Administrative action), and then close the old connection.
+
+### Keyword: “reason: {shutdown, tcp_closed}”
+
+This log indicates that the client directly closed the network connection without sending a DISCONNECT packet. The corresponding reason would be `{shutdown, normal}`, which means that the client first sent a DISCONNECT packet with a Reason Code of 0 and then closed the network connection.
+
+Therefore, when you encounter that `tcp_closed` appears in the EMQX logs, and this disconnection behavior is not what you expected, it is recommended to investigate the client's implementation for any potential issues.
+
+### Keyword: “maximum heap size reached”
+
+The appearance of this log indicates that the heap occupied by the current client process has exceeded the preset maximum value, and the client process will be forcibly killed to ensure the availability of EMQX, and avoid the unlimited growth of the memory usage of the client process and eventually lead to EMQX OOM. 
```

Comment:
```suggestion
This log appears when the heap occupied by the current client process exceeds the preset maximum value. The client process will be forcibly terminated to ensure EMQX's availability and to prevent excessive memory usage. 
```